src/docs/modules/ROOT/index.adoc

```diff
@@ -0,0 +1,35 @@
+= Welcome to the Open Manufacturing Platform - Semantic Data Structuring Documentation
+:page-layout: tiles
+
+This is even more content. 
+
+'''
+
+[.tile]
+[.icon-spec]
+--
+[.title]
+BAMM Aspect Meta Model (BAMM)
+
+[.text]
+Aspect models defined using the BAMM describe domain semantics.
+Find out how they are defined.
+
+[.link]
+xref:bamm-specification:ROOT:introduction.adoc[Specification] ⬥ xref:bamm-specification:ROOT:modeling-guidelines.adoc[Modeling Guideline]
```

Comment:
Here, the section that lives in `modeling-guidelines.adoc` is not called Modeling Guideline. I would probably remove the second link altogether.